### PR TITLE
df: add Chinese translation

### PR DESCRIPTION
**Pull request by LiLittleCat**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.zh/common/df.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+`df`
+
+- 以人类可读的形式显示所有文件系统和它们的磁盘使用情况：
+
+`df -h`
+
+- 显示包含给定文件或目录的文件系统及其磁盘使用情况:
+
+`df {{文件或目录的路径}}`
```
Comment (kbdharun) [inline pages.zh/common/df.md:16]:
```suggestion
`df {{路径/到/文件或目录}}`
```
Reference: <https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md>.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks @LiLittleCat , I just have a suggestion that needs to be addressed.

File: pages.zh/common/df.md (hunk 1 above)
Comment (LiLittleCat) [reply pages.zh/common/df.md]: Thank you for your suggestion. @kbdharun

File: pages.zh/common/df.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- 显示所有文件系统和它们的磁盘使用情况：
+
+`df`
+
+- 以人类可读的形式显示所有文件系统和它们的磁盘使用情况：
+
+`df -h`
+
+- 显示包含给定文件或目录的文件系统及其磁盘使用情况:
```
Comment (blueskyson) [inline pages.zh/common/df.md:14]:
```suggestion
- 显示包含给定文件或目录的文件系统及其磁盘使用情况：
```

**Review by blueskyson — APPROVED** [review]

**Comment by kbdharun** [comment]
@blueskyson , Check out <https://github.com/tldr-pages/tldr/pull/8391#issuecomment-1225909897>. We need to remove the co-authors from commit message it seems as @navarroaxel mentioned. There is an ongoing PR regrading updating the maintainers guide. [Ref](https://github.com/tldr-pages/tldr/pull/8398).

**Comment by blueskyson** [comment]
> @blueskyson , Check out [#8391 (comment)](https://github.com/tldr-pages/tldr/pull/8391#issuecomment-1225909897). We need to remove the co-authors from commit message it seems as @navarroaxel mentioned. There is an ongoing PR ([Link](https://github.com/tldr-pages/tldr/pull/8398)) regarding updating the maintainers guide.

I'll pay attention of that next time, thanks.

**Pull request merged**